cs/_pages/technical-details.markdown

```diff
@@ -13,58 +13,81 @@ header:
-significance in the text. The text analysis functionality has been developed separately to enable its reuse.
-This library is called **Annotace**, it is open source and can be found on [GitHub](https://github.com/kbss-cvut/annotace) as well.
+Jedna z nejdůležitějších funkcí systému TermIt je jeho schopnost analyzovat obsah dokumentů pro vyhledávání výskytu existujících pojmů a navrhování nových pojmů podle jejich významu v textu. Textovou analýzu zajišťuje samostatně vyvinutá open-source knihovna **Annotace**, kterou můžete najít v [jejím GitHub repozitáři](https://github.com/kbss-cvut/annotace).
 
 
-## Notable Libraries
+### Knihovny hodné povšimnutí
 
-TermIt uses several custom libraries and components that may be of interest for other Semantic Web developers. Here is a short list with some basic details
-and links with more details.
+TermIt používá několik knihoven vyvinutých pro účely sémantického webu a sémantických dat, a které by mohly zajímat i další vývojáře v této obasti. Jsou shrnuty v následujícím seznamu včetně zákaldního popisu a odkazu.
 
 #### JOPA
 
-[JOPA](https://github.com/kbss-cvut/jopa) (Java OWL Persistence API) is a persistence library for semantic data.
-It allows working with POJO-based domain model which is stored as RDF data in a triple store.
-Heavily inspired by JPA and its implementation, JOPA contains all the necessary bells and whistles (transactions, query result mapping, caching) to make development
-of a Semantic Web-based information system as smooth as possible.
+[JOPA](https://github.com/kbss-cvut/jopa) (Java OWL Persistence API) je perzistentní knihovna pro sémantická data. Umožuje práci s doménovým modelem založeným na POJO, který je uložen v podobě RDF dat v triple storu. Silná inspirace v JPA je vidět i v přítomnosti transakcí, mapování výsledků dotazů a cachování za účelem co nejhladšího vývoje informačního systému založeného na principech sémantického webu.
 
-There is also a [library](https://github.com/ledsoft/jopa-spring-transaction) for integrating JOPA with the declarative
-transaction management of Spring (think `@Transactional`) that is used in TermIt.
+Pro integraci JOPA s deklarativní správou transakcí Springu (myslete `@Transakčně`), která je použita v systému TermIt je zde i [tato knihovna](https://github.com/ledsoft/jopa-spring-transaction).
 
 #### JB4JSON-LD
-[JB4JSON-LD](https://github.com/kbss-cvut/jb4jsonld) (Java Binding for JSON-LD) is a library for seamless marshalling and unmarshalling of POJOs into and from JSON-LD.
-It integrates with Jackson, so the set-up for an application is a matter of just a couple of lines of code.
+[JB4JSON-LD](https://github.com/kbss-cvut/jb4jsonld) (Java Binding for JSON-LD) je knihovna pro bezešvé přiřazování POJO na JSON-LD a naopak. Integruje se s Jacksonem, takže nastavení aplikace je otázkou pouze několika řádek kódu.
```

Comment:
- ...je knihovna pro snadné mapování POJO na JSON-LD a naopak...
- Integruje se s Jacksonem, takže použití v aplikaci je otázkou...